### DISCUSSION TITLE
Add SNAC

### DISCUSSION THREAD

**Discussion by julien-blanchon** [speechbrain/speechbrain#2567] [Feature Request]
Are you interested into adding SNAC (https://github.com/hubertsiuzdak/snac) ? Under huggingface_transformers/snac.py maybe ?

**Comment by Adel-Moumen** [comment]
Would you like to do it, @julien-blanchon? This could be a nice contribution. I'm pinging @poonehmousavi and @mravanelli since they are very active on this topic these days.

**Reply by julien-blanchon** [reply]
Yep let's go. I'm trying a PR very quick ;)

**Comment by julien-blanchon** [comment]
Here you go @Adel-Moumen: https://github.com/speechbrain/speechbrain/pull/2568

**Reply by poonehmousavi** [reply]
Thanks, @julien-blanchon for this PR..it is interesting to expand our discrete audio tokenizert .. I will take a look at the PR soon
